### PR TITLE
Adding akamai edge worker instructions

### PR DESCRIPTION
**Pull request by gannaraputeja**

### CODE REVIEW

File: docs/install-traffic-capture-sensors/akamai-edge-worker.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+sidebar_position: 8
+description: Deploy Levo.ai PCAP sensor on Cloudflare Worker. Follow our detailed guide to set up and configure for enhanced API traffic capture and analysis.
+---
+
+# Akamai Edge Worker
+
+## Prerequisites
+- **Akamai DNS Configuration:** Your website’s DNS is managed through Akamai.
+- **Permissions:** You have the necessary permissions on Akamai to create. And configure edge workers to your akamai property.
+- **Levo Satellite Setup:** The Satellite has been successfully set up and is accessible via HTTPS from the EdgeWorker. Ensure that Domain of the Satellite URL is served by the Akamai platform. Please refer to the [Satellite Installation](/install-satellite) for more information.
```
Comment (VamsiNerella) [inline docs/install-traffic-capture-sensors/akamai-edge-worker.md:11]: how to ensure satellite URL is served by the Akamai platform?

File: docs/install-traffic-capture-sensors/akamai-edge-worker.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+#### Setup Akamai CLI and Edge Workers
+- Ensure you have [Akamai CLI](https://techdocs.akamai.com/edgeworkers/docs/akamai-cli) installed.
+  - On Windows you may need to add `akamai.exe` location to `Path` environment variable
+- Install edge workers using `akamai install edgeworkers`.
+- Ensure you have required authentication credentials like access token and client secret for your account. These can be found on Akamai **Identity & Access Management** page. 
+- For more details please refer to docs [Akamai Setup Authentication Credentials](https://techdocs.akamai.com/developer/docs/set-up-authentication-credentials).
+
+#### Create Edge Worker ID
```
Comment (VamsiNerella) [inline docs/install-traffic-capture-sensors/akamai-edge-worker.md:62]: where/how do they have to add/configure our instrumentation code?

File: docs/install-traffic-capture-sensors/akamai-edge-worker.md (hunk 1 above)
Comment (gannaraputeja) [reply docs/install-traffic-capture-sensors/akamai-edge-worker.md:11]: Usually done with CNAME in akamai. In our case we added a CNAME in cloudflare and created a configured in akamai using a Akamai CDN property(site dynamic accelerator).

File: docs/install-traffic-capture-sensors/akamai-edge-worker.md (hunk 2 above)
Comment (gannaraputeja) [reply docs/install-traffic-capture-sensors/akamai-edge-worker.md:62]: Following step under `Setup Edge Worker` section.

File: docs/install-traffic-capture-sensors/akamai-edge-worker.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 8
+description: Deploy Levo.ai PCAP sensor on Cloudflare Worker. Follow our detailed guide to set up and configure for enhanced API traffic capture and analysis.
```
Comment (buchi-busireddy) [inline docs/install-traffic-capture-sensors/akamai-edge-worker.md:3]: Please fix this

**Review by buchi-busireddy — APPROVED** [review]

**Pull request merged**